pkcs11/testdata/ec.cert.pem

```diff
@@ -0,0 +1,10 @@
+-----BEGIN CERTIFICATE-----
+MIIBTjCB9gIJAJ5bWBxZlNa/MAoGCCqGSM49BAMCMDAxCzAJBgNVBAYTAlVTMQsw
+CQYDVQQIDAJDQTEUMBIGA1UEAwwLZXhhbXBsZS5jb20wHhcNMjEwMzI2MTczMzEy
+WhcNMjEwNDI1MTczMzEyWjAwMQswCQYDVQQGEwJVUzELMAkGA1UECAwCQ0ExFDAS
+BgNVBAMMC2V4YW1wbGUuY29tMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE2eGr
+vNUo5MgKgeimjt3t1ZkM04DN4r7xguptdaR/9erKP0L9nODSr3fWfeKDRx/FClt/
+nZIUwV8q6JbzbuVaDjAKBggqhkjOPQQDAgNHADBEAiBSy79qkY6o1R0uZpjSTVWM
+yJiXIpOml5o+qqyGTr5lcwIgJK8CPd0rmdiUuD5G2QrK1+cZ2ZvvPwQ3Z2l/QAFk
+IjM=
+-----END CERTIFICATE-----
```

Comment:
newline

Reply:
Just for reference why a newline is recommended at end of a source file: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file